src/borg/archiver/help_cmd.py

```diff
@@ -28,15 +28,13 @@ class HelpMixIn:
 
         Starting with Borg 1.2, paths that are matched against patterns always
         appear relative. If you give ``/absolute/`` as root, the paths going
-        into the matcher will look relative like ``absolute/.../file.ext``.
-        If you give ``../some/path`` as root, the paths will look like
-        ``some/path/.../file.ext``.
+        into the matcher will look relative like ``absolute/``.
+        If you give ``../../relative`` as root, the paths will be normalized
+        as ``relative/``.
```

Comment:
I think the `file.ext` should be kept, so readers will realize this is an example full path going into the matcher.

But as we discussed, if the `...` is confusing, get rid of that.

Reply:
As you prefer, I find it more clear like that. Is "realizing this is a full path" important to understand the idea of the leading slash here ? If you confirm, then I'll readd the file.ext without dots.

Reply:
the point is that these are example paths coming from fs recursion, not something from cli args or patterns file.

Reply:
Alternatively, one could say "the paths going into the match will start with ...".

Not "will look like" without the "..." or the example path/filenanem.